src/components/form/row.vue

```diff
@@ -59,10 +59,11 @@ except according to the terms contained in the LICENSE file.
     <td v-if="showActions" class="actions">
       <template v-if="form.state !== 'closed'">
         <template v-if="form.state === 'closing'">
-          <span :title="$t('formClosingTip')">
-          <span class="icon-clock-o closing-icon"></span>
-          <span>{{ $t('formState.closing') }}</span>
+          <span aria-hidden="true" v-tooltip.sr-only>
+            <span class="icon-clock-o closing-icon"></span>
+            <span>{{ $t('formState.closing') }}</span>
           </span>
+          <span class="sr-only">{{ $t('formClosingTip') }}</span>
```

Comment:
Since screen readers will read `$t('formClosingTip')`, I don't think it's useful for them to also read `$t('formState.closing')` immediately beforehand, which is why I set `aria-hidden="true"` on the first `<span>` and didn't add a `&nbsp;` to the `.sr-only` element.